src/program.cpp

```diff
@@ -52,6 +52,32 @@ struct membuf : public std::streambuf {
         auto send = reinterpret_cast<char*>(end);
         setp(sbegin, send);
     }
+    virtual pos_type seekoff(off_type off, std::ios_base::seekdir dir,
```

Comment:
Suggest removing `virtual` (in keeping with the rest of the code base) but adding `override` since that guarantees the function is an override (which is the intended behaviour here).

Reply:
Done.